index.html

```diff
@@ -16,6 +16,8 @@
                             {% else %}
                                 <source src="/media/{{ site.lang }}/Monero_Promo.m4v">
                             {% endif %}
+                            <track label="English" kind="subtitles" srclang="en" src="/media/subtitles/en/monero-promo.vtt" default>
```

Comment:
This will create conflicts and differs from #1373. Would be better to base your PR on that one.